hermes/file_gen.go

```diff
@@ -1,28 +1,28 @@
-package main
+package hermes
+
 import (
 	"fmt"
-	// "math/rand"
 	"strconv"
 	"crypto/sha1"
 )
 type HermesFile struct {
```

Comment:
Just add some whitespace here.
```suggestion

type HermesFile struct {
```